olive/platform_sdk/qualcomm/runner.py

```diff
@@ -0,0 +1,71 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+import time
+
+from olive.common.utils import run_subprocess
+from olive.platform_sdk.qualcomm.snpe.env import SNPESDKEnv
+
+logger = logging.getLogger(__name__)
+
+
+class SDKRunner:
+    def __init__(
+        self,
+        platform,
+        cmd: str,
```

Comment:
can the runner run multiple time for different cmds? If yes, suggest to move the cmd parameter to run method.

Reply:
Moved the cmd to run.